dockerfile/aznhc-entrypoint.sh

```diff
@@ -1,5 +1,13 @@
 #! /bin/bash
 
+function collect_meta_data(){
+    # get meta data for VM and underlying host
+    vmhostname=$(hostname)
+    vmid=$( curl -H Metadata:true --max-time 10 -s  "http://169.254.169.254/metadata/instance/compute/vmId?api-version=2021-03-01&format=text")
+    vm_name=$(curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/name?api-version=2021-11-15&format=text")
```

Comment:
Nitpick: is there a reason `vm_name` uses an underscore?

Reply:
not really, let me handle

Reply:
Addressed